big_tests/tests/push_integration_SUITE.erl

```diff
@@ -310,28 +335,72 @@ send_message_to_room(Sender, Room) ->
 
 
 muclight_msg_notify_on_apns(Config, EnableOpts) ->
-    escalus:story(
+    escalus:fresh_story(
         Config, [{alice, 1}, {bob, 1}, {kate, 1}],
         fun(Alice, Bob, _Kate) ->
-            {SenderJID, DeviceToken} =
-                muclight_conversation(Config, Alice, Bob, <<"apns">>, EnableOpts),
+            RoomJID = muc_light_helper:given_muc_light_room(fresh_room_name(), Alice, [{Bob, member}]),
+            DeviceToken = enable_push_for_user(Bob, <<"apns">>, EnableOpts),
+
+            SenderJID = muclight_conversation(Alice, RoomJID, <<"Heyah!">>),
             Notification = wait_for_push_request(DeviceToken),
             assert_push_notification(Notification, <<"apns">>,
-                                     [{body, <<"Heyah!">>} | EnableOpts], SenderJID)
-
+                                     EnableOpts, SenderJID, [{body, <<"Heyah!">>}, {unread_count, 1}, {badge, 1}]),
+    ok
```

Comment:
Broken indentation. Also, why do we need to return `ok` anyway? `assert_push_notification` will itself return `ok` if everything is ok :)

Reply:
When I am experimenting with test i often add `ok` at the end to quickly comment/uncomment/change lines before;) I'll remove it

Reply:
Oh, that's actually a pretty simple trick for that, I like it :)